recipes/blend-bio/meta.yaml

```diff
@@ -0,0 +1,32 @@
+{% set version = "1.0.0" %}
+
+package:
+  name: blend-bio
+  version: 1.0.0
```

Comment:
Can you use {{ version }} here as well, to make it easier to update the recipe in the future?